docs/modules/reference/pages/daemons/daemon-config-files/provisiond.adoc

```diff
@@ -1,60 +1,55 @@
+The provision daemon scans nodes from requisition and import files, and from `newSuspect` events.
 The scanned nodes are then added to the database.
 Note that only nodes in the database are monitored by other daemons.
 
-NewSuspect events are triggered by Discovery scans, and by trapd and syslogd if those daemons are configured to create the event.
-
-We recommend that you use the {page-component-title} UI to update configuration settings.
+`newSuspect` events are generated by discovery scans.
+They can also be generated by trapd and syslogd, if those daemons are configured properly.
+We recommend that you use the {page-component-title} web UI to update configuration settings.
 
+[caption=]
 .Provisiond configuration file overview
-[options="header"]
-[cols="2,3,1,1"]
+[options="autowidth"]
 |===
-| File
-| Description
-| Reload Event
-| Restart Required
+| File  | Description   | Reload Event  | Restart Required
 
 ifeval::["{page-component-title}" == "Meridian"]
 | provisiond-configuration.xml
-| This file sets requisition definitions for defined discovery of nodes and foreign sources.
-| yes
-| yes
-endif::[]
+| Sets requisition definitions for defined discovery of nodes and foreign sources.
+| Yes
+| Yes
 
+endif::[]
 | default-foreign-source.xml
-| List of xref:reference:provisioning/detectors.adoc[detectors] and xref:reference:provisioning/policies.adoc[policies] to apply when scanning requisitioned nodes.
-Does not exist by default.
-Once these settings are modified using the UI, a `default-foreign-source.xml` file will be automatically created in `$\{OPENNMS_HOME}/etc`.
-| no
-| no
+| List of xref:provisioning/detectors.adoc[detectors] and xref:provisioning/policies.adoc[policies] to apply when scanning requisitioned nodes.
+Does not exist by default. +
+After these settings are modified via the web UI, a `default-foreign-source.xml` file is automatically created in `$\{OPENNMS_HOME}/etc`.
+| No
+| No
 
 | foreign-sources/*.xml
-| List of xref:reference:provisioning/detectors.adoc[detectors] and xref:reference:provisioning/policies.adoc[policies] to apply when scanning nodes in a requisition that matches the filename.
+| List of xref:provisioning/detectors.adoc[detectors] and xref:provisioning/policies.adoc[policies] to apply when scanning nodes in a requisition that matches the filename.
 Does not exist by default.
-| yes
-| no
+| Yes
+| No
 
 | imports/*.xml
 | One file per requisition, containing the list of nodes to import for monitoring.
-| yes
-| no
+| Yes
+| No
 |===
 
+[caption=]
 .Provisiond log file overview
-[options="header"]
-[cols="2,3"]
+[options="autowidth"]
 |===
-| File
-| Description
+| File  | Description
 
 | provisiond.log
-| The node provisioning processes are logged here.
-This also includes service detection and policy results.
+| Log of node provisioning processes.
+Also includes service detection and policy results.
 
 | GeoIpProvisioningAdapter.log
 | Logs related to the GeoIp provisioning plugin.
```

Comment:
Add cross reference to GeoIP in `reference/provisioning/adapters/geoip`?

Reply:
```suggestion
| Logs related to the xref:reference:provisioning/adapters/geoip[GeoIP provisioning plugin].
```